README.md

```diff
@@ -4,7 +4,7 @@
 ### Latest Version: 0.12.1
 ### Status: Active Development
 
-### Continuous Integration: [![Build Status](https://api.cirrus-ci.com/github/containers/libpod.svg)](https://cirrus-ci.com/github/containers/libpod)
+### Continuous Integration [![Build Status](https://api.cirrus-ci.com/github/containers/libpod.svg)](https://cirrus-ci.com/github/containers/libpod/master) [CI documentation](contrib/cirrus/README.md)
```

Comment:
Can you throw the Docs link on a new line?

Reply:
certainly.